incubator/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -73,17 +75,15 @@ spec:
               cpu: "{{.Values.Master.Cpu}}"
               memory: "{{.Values.Master.Memory}}"
           readinessProbe:
-            httpGet:
-              path: /login
+            tcpSocket:
               port: {{.Values.Master.ContainerPort}}
             initialDelaySeconds: 60
             periodSeconds: 1
             timeoutSeconds: 3
             successThreshold: 1
             failureThreshold: 10
           livenessProbe:
-            httpGet:
-              path: /login
+            tcpSocket:
```

Comment:
Something suspect here... This has been working, why do we need to change it now? I prefer to stick with http health check. Can we ping a different URL? e.g. Image/CSS/JS asset?